include/librealsense2/h/rs_internal.h

```diff
@@ -243,6 +325,19 @@ void rs2_software_sensor_add_read_only_option(rs2_sensor* sensor, rs2_option opt
  * \param[out] error  if non-null, receives any error that occurs during this call, otherwise, errors are ignored
  */
 void rs2_software_sensor_update_read_only_option(rs2_sensor* sensor, rs2_option option, float val, rs2_error** error);
+
+/**
+ * Add an option to sensor
+ * \param[in] sensor        the software sensor
+ * \param[in] option        the wanted option
+ * \param[in] min           the minimum value which will be accepted for this option
+ * \param[in] max           the maximum value which will be accepted for this option
+ * \param[in] step          the granularity of options which accept discrete values, or zero if the option accepts continuous values
```

Comment:
Floating point step is a valid option (depth units)
Zero step could be applied to const opt only, and even then it is better to confine the range by min=max

Reply:
Can you please clarify this comment?
step is a float parameter already, so floating point step is possible.
step = 0 for continuous values is lifted from deeper in the library and not something I decided
min=max for readonly options isn't desirable. e.g. ASIC_TEMPERATURE. We want to be able to give an expected range for the temperature even though the user can't set the temperature.

Reply:
min-max for const option, the sentence to be adjusted is
>or zero if the option accepts continuous values